src/brain/gocdNoDeploysAlert/index.test.ts

```diff
@@ -1,6 +1,7 @@
 import merge from 'lodash.merge';
 
-import payload from '@test/payloads/gocd/gocd-stage-building.json';
+import oldPayload from '@test/payloads/gocd/gocd-stage-building.json';
+const payload = oldPayload as GoCDResponse;
```

Comment:
Same as above, but could we also move the const out of the imports and down within the main `describe`?